CONTRIBUTING.md

```diff
@@ -55,4 +55,7 @@ non-consensually on open-source code with a variety of licenses, it's not
 obvious that we have the moral or legal right to redistribute code they
 generate.
 
-
+## Testing
```

Comment:
Item 1 here is already stated [here](https://github.com/sass/migrator/blob/main/test/migrators/README.md). It probably makes sense to add the instructions for running tests to that file too.

Reply:
Done! Thanks for the review